### PR TITLE
zoe: add version 3.3

### PR DESCRIPTION
**Pull request by toge**
### Summary
Changes to recipe:  **zoe/3.3**

#### Motivation
There are several improvements since 3.1.

#### Details
https://github.com/winsoft666/zoe/compare/v3.1...v3.3

---
- [x] Read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md)
- [x] Checked that this PR is not a duplicate: [list of PRs by recipe](https://github.com/conan-io/conan-center-index/discussions/24240)
- [x] Tested locally with at least one configuration using a recent version of Conan

### CODE REVIEW

**Review by AbrilRBS — COMMENTED** [review]
Shouldn't we reflect [winsoft666/zoe@`v3.1...v3.2`#diff-148715d6ea](https://github.com/winsoft666/zoe/compare/v3.1...v3.2#diff-148715d6ea0c0ea0a346af3f6bd610d010d490eca35ac6a9b408748f7ca9e3f4R92) in the recipe?

**Comment by toge** [comment]
@AbrilRBS 
Thank you for your reivew!
Sorry I have no idea to to follow this in the recipe.
What changes should be made?

File: recipes/zoe/all/conanfile.py [hunk 1]
```diff
@@ -63,6 +66,7 @@ def generate(self):
         tc.variables["ZOE_BUILD_SHARED_LIBS"] = self.options.shared
         tc.variables["ZOE_USE_STATIC_CRT"] = is_msvc_static_runtime(self)
         tc.variables["ZOE_BUILD_TESTS"] = False
+        tc.variables["LUNASVG_BUILD_EXAMPLES"] = False
```
Comment (AbrilRBS) [inline recipes/zoe/all/conanfile.py:69]:
```suggestion
```

**Comment by AbrilRBS** [comment]
Probably not worth fixing (Although it is unexpected for Conan users!), but the build logs have a warning of `-- Warning: Standard CMAKE_CXX_STANDARD value defined in conan_toolchain.cmake to 17 has been modified to 11 by /home/conan/workspace/prod-v2/bsr/98192/fbdbb/p/b/zoe9f2cbd1359bbe/b/src/CMakeLists.txt` due to [winsoft666/zoe@`v3.1...v3.2`#diff-1e7de1ae2d](https://github.com/winsoft666/zoe/compare/v3.1...v3.2#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR38)

**Comment by toge** [comment]
In 3.3, the library names of zoe have been reverted to these of 3.1.

File: recipes/zoe/all/conanfile.py [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 from conan.tools.microsoft import is_msvc_static_runtime
-from conan.tools.files import get, copy, apply_conandata_patches, export_conandata_patches, rmdir
+from conan.tools.files import get, copy, apply_conandata_patches, export_conandata_patches, rmdir, replace_in_file
 from conan.tools.build import check_min_cppstd
 from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
 from conan.tools.scm import Version
 import os
 
-required_conan_version = ">=1.53.0"
+required_conan_version = ">=2"
```
Comment (AbrilRBS) [inline recipes/zoe/all/conanfile.py:10]:
```suggestion
required_conan_version = ">=2.1"
```

implements was introduced in 2.0.9

**Review by AbrilRBS — APPROVED** [review]

**Review by ErniGH — APPROVED** [review]
LGTM Thanks!

**Pull request merged**